letsencrypt-auto

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/bash -e
```

Comment:
Note that `bash` is not installed on all systems by default (for instance FreeBSD AFAIR). I'd much prefer if we didn't use `-n` at all. Also, on my system, where `/bin/sh` is `dash`, `echo -n` works perfectly OK.